.github/workflows/mypy.yaml

```diff
@@ -0,0 +1,45 @@
+name: Mypy CI
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+
+concurrency:
+
+  group: "${{ github.ref }}-${{ github.head_ref }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  mypy_test :
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
```

Comment:
Why do you need this conditional? Note, I am not saying you do not need it, I genuinely do not know about the possible usefulness of that line.

Reply:
This limits this action only to pushes and PRs to `MDAnalysis/mdanalysis`, instead of forks. We have it throughout the rest of our CI, I assume as a courtesy to the GitHub servers / personal allotments.

Reply:
it's a nice way to avoid forks from running CI if they don't want to (for mypy it's not that bad, but every little helps when it comes to reducing CO2 emissions)